payment-3p/Makefile

```diff
@@ -18,14 +18,15 @@ build: _install
 		mkdir build/ ; \
 	fi
 	cp -rp src/ build/src/
+	npm i
```

Comment:
There is a target that should do this already (`_install`), [used by the build target](https://github.com/aws-samples/aws-serverless-ecommerce-platform/blob/84a322f5a19ed34eb590c6bca7337b5693aaccc0/payment-3p/Makefile#L15). Was there an issue with it?

Reply:
You're right, its redundant. Removed in next commit!